.github/workflows/prettier.yaml

```diff
@@ -0,0 +1,26 @@
+name: Apply Prettier style
+
+on:
+  push:
+    paths:
+      - "**.md"
```

Comment:
- note the job should be marked as needing write permissions https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs we've dealt with that before. 
- could be only on PRs not on main

Reply:
Added the `contents` and `pull-requests` write permissions, and rebased to force another action run to verify that those are sufficient for the action.